### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by Zheng222** [Vious/LBAM_Pytorch#1]
@Vious Hello, can you provide the pretrained model? Thank you!

**Comment by sadbhawnathakur** [comment]
@Vious  Pretrained model link is not mentioned in the description.

**Comment by Vious** [comment]
Oh, sorry for my fault, I forgot to update the link of pretrained model into the README file, for the pretrained model, please kindly refer to the link of google drive: https://drive.google.com/drive/folders/17DJEp0APDcZoWPIHwMFs78ak59pYlDPw, or Baidu Cloud: https://pan.baidu.com/share/init?surl=z9hNEpbhsK2r-TEJbYkuQw, extract code is: mvzh

**Comment by hastaluegoph** [comment]
@Vious I download the pretrained model ,  but  I found it go wrong when load the model. It may be something with the BN weights key, should I change the code?

**Comment by Vious** [comment]
@hastaluegoph I made a slightly change of the model by removing the batchnorm layer (setting bn to false) and the last tanh layer has been changed from absolute value to (tanh() + 1) / 2. I've updated the model files now or you could use the model files in the given link,  I think that would be right.

**Comment by hastaluegoph** [comment]
@Vious Thanks a lot! I modified  code under your instruction and test on places2 , but the result turn   out to be worse than the Paper's,.Should I change some parameter and retrain the model?

**Comment by Vious** [comment]
@hastaluegoph For places2 model, due to the computational limitation, I only use 10 classes for training, including apartment_building-outdoor, beach, house, ocean, sky, throne_room, tower, tundra, valley and wheat_field, you could use the above 10 validation sets for evaluating.
